ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -137,3 +137,10 @@ def arctanh(input, *, out=None):
 
 
 arctanh.unsupported_dtypes = ("float16",)
+
+
+def square(input, *, out=None):
+    return ivy.square(input, out=out)
+
+
+square.unsupported_dtypes = ("uint16", "uint64", "uint32")
```

Comment:
I think torch not handling these `uint` dtypes is alredy handled, this is not necessary :)